Task70/Task70/App/Application.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using Task70.Model;
+using Task70.Pages;
+
+namespace Task70.App
+{
+    public class Application
```

Comment:
I would say that you don't need one more layer (Application class) to implement auto tests. The better way is calling page instances directly in test classes. In case if you have more that 2 pages it will be difficult to support Application class. But for this small task it works fine.